content/blog/2019/06/2019-06-25-plugin-management-tool-alpha-release.adoc

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Plugin Management Library and CLI Tool Alpha Release"
+tags:
+- plugins
+- pluginmanagement
+- pluginmanager
+- platform-sig
+- cli
+- gsoc
+- gsoc2019
+author: stopalopa
+---
+= Plugin Management Library/CLI Tool
+
+_"Everybody is re-inventing the wheel, partially implementing the "details" of plugin management (signed metadata, artifacts checksums, plugins detached from core,...).
+It becomes obvious Jenkins should provide adequate tooling for plugin installation outside jenkins live instance."_ link:https://issues.jenkins-ci.org/browse/JENKINS-53767[JENKINS-53767]
```

Comment:
```suggestion
It becomes obvious Jenkins should provide adequate tooling for plugin installation outside a live Jenkins live instance."_ link:https://issues.jenkins-ci.org/browse/JENKINS-53767[JENKINS-53767]
```

Reply:
That was a direct quote, but I agree it was not phrased the best. I actually just updated it in Jira to match what you suggested.